extensions/skins/conversation_v1/player.html

```diff
@@ -126,6 +126,15 @@
                     (<[activeCard.interactionInstructions]>)
                     <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
                   </button>
+		</div>
+		<div ng-if="!activeCard.interactionIsInline && !canWindowFitTwoCards()"
```

Comment:
Do you think we could reduce some of the duplication by having one ng-if inside the other instead? Here, lines 131-132 are exact repeats of line 123-124.